### ISSUE TITLE
Query editor: format query feature breaks syntax of Postgres JSON operator

### ISSUE THREAD

**Issue by deed02392** [getredash/redash#5644]
### Issue Summary

Query editor: format query feature breaks syntax of Postgres JSON operator

### Steps to Reproduce

1. `WHERE keywords @> '[{"keyword":{"id":12345}}]'`
2. Valid according to: https://www.postgresql.org/docs/12/functions-json.html
3. Gets formatted as `keywords @ > '[{"keyword":{"id":12345}}]'`
4. This is invalid syntax because the `@` and `>` have become separated

### Technical details:

* Redash Version: 10.0.0-beta (92e5d78d)

**Comment by Shireenbanu** [comment]
Hi, I'm new to open source, trying to fix this issue, till now I have found the following.   

1. https://github.com/getredash/redash/blob/master/client/app/lib/queryFormat.ts This file contains the formatQuery method which returns the formatted query using sql-formatter library.
2. redash has its own modified branch of sql-formatter repo which is https://github.com/getredash/sql-formatter.git.

Can anyone please provide me further leads how to fix this issue? do I need to work on sql-formatter repo.

**Comment by deed02392** [comment]
Yes you'd need to work on the sql-formatter repo to add Postgres support.
